hydra_agent/classes_objects.py

```diff
@@ -230,19 +238,6 @@ def endpointclasses(
                 "classes", str(endpoint), node_properties)
             # set edge between the entrypoint and the class endpoint.
             self.addEdge(entrypoint_node, "has" + endpoint, class_object_node)
-
-        # for connect the nodes to endpoint which have endpoint as a property.
-        if endpoint_property_list:
-            for endpoint_property in endpoint_property_list:
-                for src_node in self.redis_graph.nodes.values():
-                    if str(endpoint_property) == src_node.alias:
-                        for endpoints in endpoint_property_list[
-                                endpoint_property]:
-                            for nodes in self.redis_graph.nodes.values():
-                                if endpoints == nodes.alias:
-                                    self.addEdge(
-                                        src_node,
-                                        "has_endpoint_property",
-                                        nodes)
-                                    break
-                        break
+            node_alias[str(endpoint)] = class_object_node
+            endpoint_property_list[endpoint] = property_list
```

Comment:
Repetition of this line. see line 232.

Reply:
@sandeepsajan0, I think I should close this PR (as @Mec-iS stated in #83)

Reply:
@ShravanDoda  You can close it. And try to do it in a better way as @Mec-iS suggest.